front/components/assistant/AssistantPicker.tsx

```diff
@@ -0,0 +1,102 @@
+import {
+  Button,
+  DropdownMenu,
+  MagnifyingGlassStrokeIcon,
+  PlusIcon,
+  RobotIcon,
+  WrenchIcon,
+} from "@dust-tt/sparkle";
+import Link from "next/link";
+import { useState } from "react";
+
+import { AgentConfigurationType } from "@app/types/assistant/agent";
+import { WorkspaceType } from "@app/types/user";
+
+export function AssistantPicker({
+  owner,
+  assistants,
+  onItemClick,
+  pickerButton,
+  manageButtons,
```

Comment:
the name is a bit confusion especially with a pickerButton argument next to it.

Let's call it `showBuilderActions`